daprdocs/content/en/roadmap/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+type: docs
+title: "Dapr Roadmap"
+linkTitle: "Dapr Roadmap"
+description: "The Dapr Roadmap is a tool to help with visibility into investments across the Dapr project"
+weight: 200
+no_list: true
+---
+
+### [Dapr Roadmap](https://aka.ms/dapr/roadmap)
+
+Dapr encourages the community to help with prioritization. Please vote by adding a 👍 on the GitHub issues for the feature capabilities you would most like to see Dapr support. This will help the Dapr maintainers understand which features will provide the most value.
+
+Dapr welcomes contributions from the community. If there are features on the roadmap that you are interested in contributing to, please comment on the GitHub issue and include your solution proposal, if available.
+
+<br />
+The Dapr Roadmap progresses through the following four stages:
+
+<div class="card-deck">
```

Comment:
Let's update Docsy and use the card shortcode

Reply:
Updated